net-libs/tox/tox-9999.ebuild

```diff
@@ -44,19 +45,18 @@ src_configure() {
 	local mycmakeargs=(
 		-DAUTOTEST=$(usex test)
 		-DBOOTSTRAP_DAEMON=$(usex daemon)
+		-DBUILD_MISC_TESTS=$(usex test)
 		-DBUILD_TOXAV=$(usex av)
 		-DDHT_BOOTSTRAP=$(usex dht-node)
 		-DENABLE_SHARED=ON
 		-DENABLE_STATIC=$(usex static-libs)
 		-DMUST_BUILD_TOXAV=$(usex av))
 	if use test; then
 		mycmakeargs+=(
-			-DBUILD_AV_TEST=$(usex av)
 			-DTEST_TIMEOUT_SECONDS=120
 			-DUSE_IPV6=$(usex ipv6))
```

Comment:
You only want IPv6 if tests?

Reply:
The ipv6 cmake option is only used to "Use IPv6 in tests". See https://github.com/TokTok/c-toxcore/blob/master/INSTALL.md if interested.